test/remote/gateways/remote_adyen_test.rb

```diff
@@ -814,6 +814,39 @@ def test_failed_refund
     assert_equal 'Original pspReference required for this operation', response.message
   end
 
+  # Adyen split refund needs to have account configured/set up for this to work.
+  # Will assert failure to demonstrate that we are sending over the splits object
+  def test_successful_request_to_adyen_with_split_type
+    options = {
+      reference: '345123',
+      email: 'john.smith@test.com',
+      ip: '77.110.174.153',
+      shopper_reference: 'John Smith',
+      billing_address: @us_address,
+      order_id: '123',
+      stored_credential: { reason_type: 'unscheduled' },
+      splits: [
+        {
+          'amount' => {
+            'value' => 100
+          },
+          'type' => 'BalanceAccount',
+          'account' => '123345566',
+          'reference' => 'QSSF7B4X79BBRX65'
+        },
+        {
+          'type' => 'PaymentFee',
+          'account' => '123345566'
+        }
+      ]
+    }
+
+    purchase = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success purchase
+    assert refund = @gateway.refund(@amount, purchase.authorization, options)
+    assert_failure refund
```

Comment:
If we can't get a passing remote test, I'm wondering if a unit test makes more sense to verify that we're still passing all the appropriate `splits` fields, regardless of the `amount` being present or not?